packages/core/src/local-admin-api.ts

```diff
@@ -32,7 +38,36 @@ export class LocalAdminApi implements AdminApi {
 
   async startIndexingModelData(modelData: Array<ModelData>): Promise<void> {
     await this.indexApi.indexModels(modelData)
-    await this.syncApi.startModelSync(modelData.map((idx) => idx.streamID.toString()))
+
+    if (this.syncApi.enabled) {
+      const models = await Promise.all(modelData.map(({ streamID }) => this.loadStream(streamID)))
+
+      const oldestModel = models.reduce((oldestModel, currentModel) => {
```

Comment:
can we make a helper method for figuring out what should be the start block we start the sync at?